xdocs/changes.xml

```diff
@@ -105,6 +105,11 @@ Summary
   <li><pr>5710</pr>Add GitHub Issue templates</li>
   <li><pr>5713</pr>Update Spock to 2.2-groovy-3.0 (from 2.1-groovy-3.0)</li>
   <li><issue>5718</issue>Update Apache commons-text to 1.10.0 (from 1.9)</li>
+  <li><issue>5725</issue>Update Jackson Core to 2.13.4 (from 2.13.2)</li>
```

Comment:
It probably doesn't matter much, but the correct markup for a pull request is `<pr>5725</pr>`.

Apart from this, thanks for the well prepared PR.

Reply:
mmh, do not know how i missed that. Thanks for pointing out.

btw - regenerating the checksum.xml file gradle added three dependencies from Kotlin too. I have not pushed them as  there are no changes in regard to this dependencies from my side. But it might be needed to re-evalute them.

These are the "groups:modules" it want to add to checksums file:
- group='io.github.microutils' module='kotlin-logging'
- group='org.jetbrains.kotlinx' module='atomicfu'
- group='org.jetbrains.kotlinx' module='kotlinx-coroutines-core'